src/notification_center/traits.rs

```diff
@@ -1,3 +1,5 @@
+// TODO: This should be moved elsewhere
```

Comment:
I don't really understand why this trait is under `notification_center`. It also isn't very obvious how this should be used, except for the examples.

Reply:
Yeah, this is... ancient and really just dates back to when an early version of this was used in [subatomic](https://github.com/secretkeysio/subatomic/). It can def be moved.